les/metrics.go

```diff
@@ -112,6 +112,8 @@ var (
 
 	requestRTT       = metrics.NewRegisteredTimer("les/client/req/rtt", nil)
 	requestSendDelay = metrics.NewRegisteredTimer("les/client/req/sendDelay", nil)
+
+	clientDiscoveredNodesCounter = metrics.NewRegisteredCounter("les/client/discovered", nil) // Counter for discovered nodes
```

Comment:
It seems like a meter might be better than a count here, because if I understand correctly, counters do not report rates over a given time period, but meters do.

Reply:
When using something like prometheus, you report only the counter. (the simplest thing you can do) and leave those problems to the monitoring engine.

In PromQL you would do: `rate(les_client_discovered[1m])`